aten/src/ATen/native/FunctionOfAMatrixUtils.cpp

```diff
@@ -55,7 +56,7 @@ Tensor& _compute_linear_combination_out(const Tensor& input, const Tensor& coeff
   // output.sizes() = [m, 1 (instead of n), ...].
```

Comment:
From the doc, if coefficients is m by n, the input is `[n, ...]`, the result is `[m, ...]`. If all of a sudden n is zero while the rest is not, we could do what matmul does when reducing over the zeroth dim.